apps/src/applab/designElements/screen.jsx

```diff
@@ -171,7 +194,38 @@ export default {
     // Properly position existing screens, so that canvases appear correctly.
     element.style.position = 'absolute';
     element.style.zIndex = 0;
-
     element.setAttribute('tabIndex', '1');
+
+    if (!element.getAttribute('data-theme')) {
+      element.setAttribute(
+        'data-theme',
+        applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
+      );
+    }
+
+    if (element.style.backgroundColor === '') {
+      element.style.backgroundColor = this.themeValues.backgroundColor[
+        applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
+      ];
+    }
+  },
+  readProperty: function(element, name) {
```

Comment:
This shouldn't be a switch statement. Use an if statement instead (same for onPropertyChange)